.github/workflows/ci.yml

```diff
@@ -34,7 +34,7 @@ jobs:
           audit_token: 'fake_token'
           home: ${{ github.workspace }}/conan_home
           cache_packages: 'true'
-          python_version: '3.12'
+          python_version: '3.10'
```

Comment:
Aren't we deprecating this? Why is it used?

Reply:
Only to see the deprecation message when testing, so we can check it as validation.